scripts/generate-markdown/markdown.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const fs = require('fs');
+const markdownMagic = require('markdown-magic'); // eslint-disable-line import/no-extraneous-dependencies
+
+const header = `| Terra Package      | Version | Status | Dependencies |
+|--------------------|---------|--------|--------------|
+`;
+
+const config = {
+  transforms: {
+    // see: https://github.com/camacho/markdown-magic-subpackage-list
+    SUBPACKAGELIST(content, _options, configuration) {
+      const packagesDir = path.resolve(
+        path.dirname(configuration.originalPath), './packages',
+      );
+
+      return header + fs
+        .readdirSync(packagesDir)
+        .map((packageName) => {
```

Comment:
We should probably still check for a package.json in the off chance a package gets moved

Reply:
Added the check back in, theoretically: [ef5e90f](https://github.com/cerner/terra-clinical/pull/445/commits/ef5e90f32fb556561d92e97d2101baca9d8b08bf)